samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -159,17 +160,20 @@
     "CompletionTokenLimit": 4096,
     "ResponseTokenLimit": 1024,
 
-    "SystemDescription": "This is a chat between an intelligent AI bot named Copilot and {{$audience}}. SK stands for Semantic Kernel, the AI platform used to build the bot. The AI was trained on data through 2021 and is not aware of events that have occurred since then. It also has no ability to access data on the Internet, so it should not claim that it can or say that it will go and look things up. Try to be concise with your answers, though it is not required. Knowledge cutoff: {{$knowledgeCutoff}} / Current date: {{TimeSkill.Now}}.",
-    "SystemResponse": "Provide a response to the last message. Do not provide a list of possible responses or completions, just a single response. If it appears the last message was for another user, send [silence] as the bot response.",
+    "SystemDescription": "This is a chat between an intelligent AI bot named Copilot and one or more participants. SK stands for Semantic Kernel, the AI platform used to build the bot. The AI was trained on data through 2021 and is not aware of events that have occurred since then. It also has no ability to access data on the Internet, so it should not claim that it can or say that it will go and look things up. Try to be concise with your answers, though it is not required. Knowledge cutoff: {{$knowledgeCutoff}} / Current date: {{TimeSkill.Now}}.",
+    "SystemResponse": "Either return [silence] or provide a response to the last message. If you provide a response do not provide a list of possible responses or completions, just a single response. ONLY PROVIDE A RESPONSE IF the last message WAS ADDRESSED TO THE 'BOT' OR 'COPILOT'. If it appears the last message was not for you, send [silence] as the bot response.",
     "InitialBotMessage": "Hello, nice to meet you! How can I help you today?",
     "KnowledgeCutoffDate": "Saturday, January 1, 2022",
 
+    "SystemAudience": "Below is a chat history between an intelligent AI bot named Copilot with one or more participants.",
+    "SystemAudienceContinuation": "Using the provided chat history, generate a list of names of the participants of this chat. Do not include 'bot' or 'copilot'.The output should be a single rewritten sentence containing only a comma separated list of names. DO NOT offer additional commentary. DO NOT FABRICATE INFORMATION.\nParticipants:",
```

Comment:
How important is it to know the audience of the chat history? This adds another round trip and seems expensive in terms of token costs since each trip passes the chat history again.

Reply:
It's quite important to allow the bot to know the audience. With the new user schema you are working on, we should be able to remove this completely in the future.